lib/harvester.js

```diff
@@ -353,6 +359,20 @@ Harvester.prototype.exportRoles = function () {
     return roles;
 };
 
+Harvester.prototype.exportPermissions = function () {
+    var permissions = [];
```

Comment:
I was imagining that you'd build the list of permissions as you were registering routes, so that this method would be O(1), but this approach also makes sense since it means we don't have to register routes to know which permissions are exportable.  Nice.